src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -41,7 +43,17 @@
 @snippet generateBody(service)
     {@alwaysImport()}
 
-    // TODO generate service doc
+    /**
+    {@docLinesFromStr(serviceDocString(service))}
```

Comment:
Since it is just one line, we can just have:
* Service Description:  {@context.getDescription(protoElement)}